### PR TITLE
Adding debian task and schema

### PR DESCRIPTION
**Pull request by Ara4Sh**
Adding Task.Os.Install.Debian

### CODE REVIEW

File: lib/task-data/schemas/install-ubuntu.json [hunk 1]
```diff
@@ -68,6 +68,12 @@
         },
         "progressMilestones": {
             "$ref": "types-installos.json#/definitions/ProgressMilestones"
+        },
+        "timezone": {
+            "$ref": "types-installos.json#/definitions/Timezone"
+        },
+        "ntp": {
+            "$ref": "types-installos.json#/definitions/Ntp"
```
Comment (pengz1) [inline lib/task-data/schemas/install-ubuntu.json]: install-debian.json is exactly the same with install-ubuntu.json, you should remove install-ubuntu.json and reference only install-debian.json in install debian/ubuntu tasks.

File: lib/task-data/tasks/install-debian.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+// Copyright 2017, EMC, Inc.
```
Comment (pengz1) [inline lib/task-data/tasks/install-debian.js]: EMC => Dell EMC

File: lib/task-data/schemas/install-debian.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
+{
+    "copyright": "Copyright 2017, EMC, Inc.",
```
Comment (pengz1) [inline lib/task-data/schemas/install-debian.json]: EMC => Dell EMC

**Comment by Ara4Sh** [comment]
I close this PR to add changes and create only one  Task.Os.Install.Debian to install both ubuntu and debian.

**Pull request closed without merging**